stylelint-polaris/README.md

```diff
@@ -465,122 +465,200 @@ Disallows use of legacy mixin map data.
 
 ### Depth
 
-#### depth/declaration-property-value-disallowed-list
-
-```diff
-
-```
-
 #### depth/declaration-property-unit-disallowed-list
 
-```diff
-- box-shadow: inset 0 0 0 1px var(--p-border-subdued);
-```
+Try to use the [Polaris depth tokens](https://polaris.shopify.com/tokens/z-index) before creating your own custom styles.
```

Comment:
I repeat wording sometimes in this readme so that if we link to a specific rule from the console that users wouldn't have to scroll up to the overall category to read critical information